pymc_marketing/mmm/base.py

```diff
@@ -665,7 +665,7 @@ def plot_budget_scenearios(
         standardize_scenarios_dict_keys(base_data, ["contribution", "budget"])
 
         fig, axes = plt.subplots(nrows=1, ncols=2, figsize=(15, 6))
-        scenarios = [base_data] + list(scenarios_data)
+        scenarios = [base_data, *list(scenarios_data)]
```

Comment:
Was this whole method not tested to begin with?

Reply:
Why? that's equivalent code

Reply:
This was an automatic ruff refactor 😄

Reply:
Says there is a code coverage difference

Reply:
Ah... maybe it's only used in NB plots?

Reply:
I think so :)